packages/device_info_plus/device_info_plus/lib/src/model/android_device_info.dart

```diff
@@ -89,6 +90,10 @@ class AndroidDeviceInfo extends BaseDeviceInfo {
   /// https://developer.android.com/reference/android/os/Build#PRODUCT
   final String product;
 
+  /// The name of the device.
+  /// https://developer.android.com/reference/android/os/Build#NAME
```

Comment:
this documentation is incorrect, it should link to `https://developer.android.com/reference/android/provider/Settings.Global#DEVICE_NAME`